lib/client.js

```diff
@@ -311,7 +341,48 @@ Client.prototype = {
     }.bind(this));
   },
 
+  /// #### client.instance(instanceGuid)
+  ///
+  /// Create an Client wrapper for another app instance.
+  ///
+  /// ##### Arguments
+  ///
+  ///   * `instanceGuid` the GUID of the other instance of your app
+  ///
+  /// ##### Returns
+  ///
+  /// A [Promises/A+](https://promisesaplus.com) conformant `promise` object.
+  ///
+  /// ```javascript
+  /// var client = ZAFClient.init();
+  ///
+  /// client.get('instances').then(function(data) {
+  ///   var guid = Object.keys(data.instances).filter(function(guid) {
```

Comment:
what about the data hanging off objects? isn't that the general idea of a data-node like structure? I mean, the alternative is having an array that you have to look inside each and grab about the `instanceGuid` property...